src/MarkDown/Tos.md

```diff
@@ -0,0 +1,63 @@
+
+## Our Services
+
+The information provided when using the Services is not intended for distribution to or use by any person or entity in any jurisdiction or country where such use would be contrary to law or regulation or which subject us to any registration requirement within such jurisdiction or country. Accordingly, those persons who choose to access the Services in other locations do so on their own initiative and are soley responsible for compliance with local laws, if and to the extent local laws are applicable.
+
+MovieMentor should not be used by government agencies where secrecy is required. MovieMentor is also not a medical application; while all user data is stored on-device, we are not beholdent to the Health Insurance Portability and Accountability Act (HIPAA).
+
+## Intellectual Property Rights
+
+The source code of the Apps and the Site are open-sourced under the MIT License. Movie data is provided by The Movie Database (TMDB) and streaming data is provided by JustWatch. We may link to such services to provide additional functionality to you (the User). All user-submitted content is to be irreversibly and perpetually licensed under a CC-0 license; contributors waive their copyrights under the terms of the CC-0 license.
+
+You are required to own the copyright of any and all submitted content. If you believe any material available on or through the Services violates any copyrights you own or control, please send takedown notices to moviementor.copyright@divi.sh
+
+MovieMentor, its logo, and the MovieMentor TV Creature are the shared intellectual property of the Developers.
+
+The Developers are to abide by security best practices and to keep user data safe to our best ability.
+
+## User Registration
+
+By creating an account for the Services, you agree that submitted information is accurate, kept up-to-date, and does not perform the following:
+
+- Perform any actions that may lead us to violate our contractual agreements with The Movie Database, JustWatch, Amazon Web Services, Cloudflare, GitHub, or any other dependent parties, including data scraping or unauthorized use of our Services.
+- Trick, defraud, or mislead us and other users for any reason.
+- Circumvent, disable, or interfere with security-related features,  in line with the Computer Fraud and Abuse Act (CPAA).
+- Upload and transmit viruses, Trojan horses, trackers, spyware, other malware, or other illicit material, including excessive use of capital letters, abuse of Unicode characters, and other "spammy" content (posting of repetitive and/or derivative content). 
```

Comment:
I think spam should be used instead of "spammy" for professionalism

Reply:
fair.